src/ess/diffraction/correction.py

```diff
@@ -62,8 +73,10 @@ def normalize_by_monitor(
 
 
 def normalize_by_vanadium(
-    data: sc.DataArray, *, vanadium: sc.DataArray, edges: sc.Variable
-) -> sc.DataArray:
+    data: MergedPixels[SampleRun],
+    vanadium: MergedPixels[VanadiumRun],
```

Comment:
I think in powder, the reduction (pixel merging) is referred to as "focussing", I don't know if that name should be adopted here?

Reply:
Probably